torchvision/prototype/datasets/_builtin/pcam.py

```diff
@@ -40,15 +44,25 @@ def __iter__(self) -> Iterator[Tuple[str, io.IOBase]]:
 _Resource = namedtuple("_Resource", ("file_name", "gdrive_id", "sha256"))
 
 
-class PCAM(Dataset):
-    def _make_info(self) -> DatasetInfo:
-        return DatasetInfo(
-            "pcam",
-            homepage="https://github.com/basveeling/pcam",
-            categories=2,
-            valid_options=dict(split=("train", "test", "val")),
-            dependencies=["h5py"],
-        )
+@register_info(NAME)
+def _info() -> Dict[str, Any]:
+    return dict(
+        categories=[0, 1],
```

Comment:
Categories should be strings

```suggestion
        categories=["0", "1"],
```

Reply:
Why? That sounds overkill for non-labeled classes which are usually treated as ints in other popular frameworks

Reply:
The the naming scheme a "label" is the integer representation while "category" is the human-readable string representation. That is the same what `categories=2` did before:

https://github.com/pytorch/vision/blob/08cc9a7fb5baa9655fa5d2dd44a6fcfcbe48a1c7/torchvision/prototype/datasets/utils/_dataset.py#L44-L45

So we either should stick to this scheme and make the categories strings or drop the categories. I agree, the latter might be the better option for datasets that don't have human readable categories.

Reply:
Let's just mark this as a TODO for later. For now we can assume that all datasets have a `categories` info. I guess it's still useful so as to know the number of classes